### ISSUE TITLE
about the inception layer

### ISSUE THREAD

**Issue by dingchenwei** [Cysu/dgd_person_reid#31]
Hi Cysu,
    I'm trying to reproduce your paper with keras because I am not familiar with caffe. My question is: Why the first dimension of the output size of inception(4a) is 256? It seems the number should be (32 + 32 +32 + 32) = 128. I don't understand what the the word "double" in "double #3x3 reduce" means in the paper. Sorry for the naive question but...Can you help me?

**Comment by Cysu** [comment]
Oh. That's my bad. There were typos in the paper. The number of filters in the inception 4,5,6 blocks should be 64, 128, 256, respectively.

The "double #3x3 reduce" actually adopts from the Batch Normalization [paper](https://arxiv.org/pdf/1502.03167.pdf), which replaces the original 5x5 conv with two 3x3 convs.

**Comment by dingchenwei** [comment]
Thanks!
